components/Tabs/src/MDCTabBar.m

```diff
@@ -374,6 +376,20 @@ - (void)setBottomDividerColor:(UIColor *)bottomDividerColor {
   }
 }
 
+// UISemanticContentAttribute was added in iOS SDK 9.0 but is available on devices running earlier
+// version of iOS. We ignore the partial-availability warning that gets thrown on our use of this
+// symbol.
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wpartial-availability"
+- (void)mdf_setSemanticContentAttribute:(UISemanticContentAttribute)semanticContentAttribute {
```

Comment:
Would it work to declare this in the header and mark it with `NS_AVAILABLE_IOS(9_0)` ?

Reply:
I needed to implement the getter as well, since this would otherwise synthesize and unused ivar.

But it seems to work.

PTAL.

Reply:
Reverted those changes.